### PR TITLE
User signup + error dispatching in thunks to toast

### PR DESCRIPTION
**Pull request by ShelOrock**
Symbols and thunk messages.

### CODE REVIEW

**Review by nries1 — COMMENTED** [review]
Amazing! Both forms look great!
The add product form lets me add a product and toasts me.
I didn't test an error in add product because the front end validation is too smart!
Is the sign up form currently posting to users? Or still setting that up?

**Comment by ShelOrock** [comment]
let me through b4 travis ci 👀

File: app/components/Toasts.js [hunk 1]
```diff
@@ -3,11 +3,13 @@ import Toast from 'react-bootstrap/Toast';
 import { connect } from 'react-redux';
```
Comment (nries1) [inline app/components/Toasts.js:3]: Awesome Thunk component!

File: app/components/root.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ import ShoppingCart from './ShoppingCart';
 import Checkout from './Checkout';
```
Comment (nries1) [inline app/components/root.js:18]: Nice! glad this can go in root.

File: app/redux/constants.js [hunk 3]
```diff
@@ -1,17 +1,17 @@
-export const SET_PRODUCTS = 'SET_PRODUCTS';
+export const SET_PRODUCTS = Symbol('SET_PRODUCTS');
```
Comment (nries1) [inline app/redux/constants.js:1]: Thanks. Looks like you had some fun here.

File: app/redux/thunks/OrderDetailsThunks.js [hunk 4]
```diff
@@ -18,8 +26,20 @@ export const fetchOrderDetails = orderId => {
 export const postOrderDetails = (orderId, orderDetailId, orderDetail) => {
     return dispatch => {
         return axios.post(`/api/orders/${ orderId }/orderDetails/${ orderDetailId }`, orderDetail)
-        ,then(() => dispatch(fetchOrderDetails(orderId)))
-        .catch(e => console.error('Error creating order details', e))
+        ,then(() => {
```
Comment (nries1) [inline app/redux/thunks/OrderDetailsThunks.js]: should be **.**then ?

**Review by nries1 — COMMENTED** [review]
This is great. I think there's just one syntax error that I marked in comments on line 29 of thunks/OrderDetails. Other than that, this looks good to merge.

**Review by nries1 — APPROVED** [review]
cool cool

**Pull request merged**